functions/src/Services/Elections.ts

```diff
@@ -11,4 +13,18 @@ export const openElections = () => {
 		})
 		.then(() => Promise.resolve())
 		.catch(error => Promise.reject(error));
+};
+
+export const addPosition = (position: Position) =>{
```

Comment:
should u put this in a position service file not elections

Reply:
ok sounds good